ragna/deploy/_cli/config.py

```diff
@@ -252,19 +252,6 @@ def _wizard_common() -> Config:
         qmark=QMARK,
     ).unsafe_ask()
 
-    if questionary.confirm(
```

Comment:
Does this mean that it is no longer possible to configure the db location from the wizard? If so, what is the rationale for this?